test_nnf.py

```diff
@@ -733,6 +733,35 @@ def test_iff(a: nnf.NNF, b: nnf.NNF):
                 c.satisfied_by(model))
 
 
+@given(NNF())
+def test_forget(sentence: nnf.NNF):
+    # Test that forgetting a backbone variable doesn't change the theory
+    T = sentence & Var('added_var')
+    assert sentence.equivalent(T.forget({'added_var'}))
+
+    # Assumption to reduce the time in testing
+    assume(sentence.size() <= 15)
```

Comment:
I think it's best to put this right at the start of the test. The first assertion is already expensive.